modules/apps/headless/headless-asset-library/headless-asset-library-impl/src/main/java/com/liferay/headless/asset/library/internal/resource/v1_0/AssetLibraryResourceImpl.java

```diff
@@ -130,10 +130,10 @@ public Page<AssetLibrary> getAssetLibrariesPage(
 			throw new UnsupportedOperationException();
 		}
 
-		OrderByComparator<Group> assetLibraryOrderByComparator = null;
+		OrderByComparator<Group> groupOrderByComparator = null;
 
-		if (sorts != null) {
-			assetLibraryOrderByComparator = new AssetLibraryOrderByComparator<>(
+		if ((sorts != null) && (sorts.length > 0)) {
+			groupOrderByComparator = new GroupAssetLibraryComparator<>(
```

Comment:
I think Brian wants you to inline the class, not rename it.

Reply:
> I can not inline it:
> should be null or a
> 
> OrderByComparator<Group> groupOrderByComparator =
> 			new GroupNameComparator(true)
> See next commit

I changed the order of the commit to be more clear

I do not know if this comments are sent... 

to me it looks like pending